include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -602,53 +602,47 @@ namespace VectorTools
     Vector<number> cell_data_1(dof_1.get_fe().dofs_per_cell);
     Vector<number> cell_data_2(dof_2.get_fe().dofs_per_cell);
 
-    std::vector<short unsigned int> touch_count(
-      dof_2.n_dofs(), 0); // TODO: check on datatype... kinda strange (UK)
+    // Prevent multiple processing of the same dof.
+    std::vector<bool> touched_dofs(dof_2.n_dofs(), false);
+
     std::vector<types::global_dof_index> local_dof_indices(
       dof_2.get_fe().dofs_per_cell);
 
-    typename DoFHandler<dim, spacedim>::active_cell_iterator h =
```

Comment:
Why do you remove the second template argument here and below?

Reply:
The typos have been fixed.